### ISSUE TITLE
Multiblock furniture hitboxes do not execute commands correctly (1.16.5)

### ISSUE THREAD

**Issue by Mike5357** [PluginBugs/Issues-ItemsAdder#1867]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same issue was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already asked on the **#💬ia-community-help** channel on **Discord** to know if anyone already has a solution for the issue.

### Discord tag (optional)

Mike🎤#5357

### What happened?

As reported in Discord, there is an issue where I have a 3x3x1 furniture object with a command attached to interacting with it.
When I first place the item, I am able to interact on all 9 barrier blocks and the commands are executed, but when the server reboots only the middle 2x1 where the armor stand is is interactable. The other 7 blocks lose their interactable functionality.

Breaking and replacing it returns the functionality (as shown in video attached at bottom), but only until the next reboot

### Steps to reproduce the issue

1. Create an armor_stand furniture entity with a solid 3x3 hitbox, and attach a command to interacting with it (items yml attached below)
2. Place the furniture and interact with it; all 9 barrier blocks will execute the command
3. Restart the server.
4. Interact with the furniture again; only the middle 2x1 is still interactable. The other 7 blocks will not execute the command unless you break the furniture and re-place it again.

### Server version

CraftBukkit version 3096a-Spigot-9fb885e-af1a232 (MC: 1.16.5) (Implementing API version 1.16.5-R0.1-SNAPSHOT)

### ItemsAdder Version

ItemsAdder version 3.2.3

### ProtocolLib Version

My attached log shows version 4.8.0, but I have also tried with 5.0.0-SNAPSHOT-b575 and -b577 and the issue is present in all of them.

### LoneLibs Version

LoneLibs version 1.0.20

### FULL server log

https://mclo.gs/p01vk6Y

### Problematic items yml configuration file (optional)

```shell
info:
  namespace: "thenodemc"                 
items:
  questboard:
    display_name: "Quest Board"
    permission: thenodemc.admin
    resource:
      material: PAPER
      generate: false
      model_path: decorations/questboard
    behaviours:
      furniture:
        entity: armor_stand
        small: false
        solid: true
        fixed_rotation: true
        placeable_on:
          walls: false
          ceiling: false
          floor: true
        hitbox:
          length: 1
          width: 3
          height: 3
          length_offset: 0
          width_offset: 0
          height_offset: 0
    events:
      placed_armorstand:
        interact:
          execute_commands:
            the_first_command:
              command: me interacted with the quest board!
              as_console: false
```

### Screenshots/Videos (you can drag and drop files or paste links)

https://user-images.githubusercontent.com/12630858/180491953-bef849a4-3403-47e4-9678-2863080f5641.mp4

**Comment by LoneDev6** [comment]
Thanks, this will be looked into as soon as possible

**Comment by Mike5357** [comment]
Unfortunately still having this issue after updating to 3.2.3c

**Comment by Mike5357** [comment]
The issue actually seems to be worse now- when breaking the furniture after a reboot, it leaves behind the additional barrier blocks instead of removing them (on top of the barrier blocks not executing the commands)

**Comment by LoneDev6** [comment]
I see you're using Spigot 1.16.5, please update to the latest Paper 1.16.5, I'm sure the problem occurs because the Spigot version you're using is old.
https://api.papermc.io/v2/projects/paper/versions/1.16.5/builds/794/downloads/paper-1.16.5-794.jar
Let me know.

**Comment by Mike5357** [comment]
> I see you're using a Spigot 1.16.5 build. Please update to the latest Paper 1.16.5, I'm sure the problem occurs because the Spigot build you're using is old. https://api.papermc.io/v2/projects/paper/versions/1.16.5/builds/794/downloads/paper-1.16.5-794.jar Let me know.
Unfortunately I am not able to do so, as we are using Arclight which does not have Paper support.

**Comment by Mike5357** [comment]
> I see you're using a Spigot 1.16.5 build. Please update to the latest Paper 1.16.5, I'm sure the problem occurs because the Spigot build you're using is old. https://api.papermc.io/v2/projects/paper/versions/1.16.5/builds/794/downloads/paper-1.16.5-794.jar Let me know.

Unfortunately I am not able to do so, as we are using Arclight which does not have Paper support.

**Comment by LoneDev6** [comment]
Arclight might not have this particular event which is mandatory for furniture to work: `EntitiesLoadEvent`.
Or it simply might have some sort of trickery which causes this issue, since ItemsAdder should work fine even without the `EntitiesLoadEvent` event as it waits 5 seconds on each chunk load to make sure that entities are available, then apply the hitboxes.

So I think it's an Arclight issue.
If you want you can setup a test server to test if the issue occurs also on Paper/Spigot 1.16.5, but I did some tests and it doesn't.

Have a nice day

**Comment by Mike5357** [comment]
> Arclight might not have this particular event which is mandatory for furniture to work: `EntitiesLoadEvent`. Or it simply might have some sort of trickery which causes this issue, since ItemsAdder should work fine even without the `EntitiesLoadEvent` event as it waits 5 seconds on each chunk load to make sure that entities are available, then apply the hitboxes.
> 
> So I think it's an Arclight issue. If you want you can setup a test server to test if the issue occurs also on Paper/Spigot 1.16.5, but I did some tests and it doesn't.
> 
> Have a nice day

@LoneDev6 I noticed that the lastest changelog mentions "fixed Arclight 1.15-1.16 compatibility", however this issue is still present. I tested on Paper and this issue is not present, but is on regular Spigot (which is why its also present in Arclight- it does not have Paper.)

**Comment by LoneDev6** [comment]
"fixed Arclight 1.15-1.16 compatibility" was referring to another issue.
Seems a Spigot bug then, I can't see how `EntitiesLoadEvent` is not triggered even on the official Spigot release. Maybe you have to download a more recent Spigot build?
